includes/wc-term-functions.php

```diff
@@ -791,3 +791,34 @@ function wc_get_product_visibility_term_ids() {
 		)
 	);
 }
+
+/**
+ * Get a thumbnail image for term.
+ *
+ * The returned variable is filtered by woocommerce_get_term_thumbnail_url filter to
+ * allow 3rd party customisation.
+ *
+ * @param array|int $term Term object of the target term, or term ID.
+ * @return string Final image url.
+ */
+function wc_get_term_src_url( $term, $image_id = false, $size = false, $placeholder_image = false ) {
```

Comment:
Can we name this `wc_get_term_image_url` so it's more similar to `wp_get_attachment_image_url`. Args should simply be $term_id and $size. The others are not needed.

You may need to add an `wc_get_term_image_id` also for code which needs ID rather than src.

Reply:
Mike, you are free to modify this as you think it will be better. 
Or I have to modify it?

Reply:
What if we leave the other arguments? They are important, because if you carefully view  my other commited files, there are cases when it RELIES on those parameters.